backend/src/zac/contrib/dowc/api.py

```diff
@@ -73,6 +74,26 @@ def create_doc(
     return factory(DowcResponse, response), status_code
 
 
+@optional_service
+def get_open_documenten(user: User, referer: str) -> List[Optional[DowcResponse]]:
+    client = get_client(user)
+    operation_id = "documenten_list"
+    url = get_operation_url(client.schema, operation_id)
+    url = furl(url).add(
+        {
+            "purpose": DocFileTypes.write,
+            "info_url": referer,
+        }
+    )
+    try:
+        response = client.request(
+            url.url, operation_id, method="GET", expected_status=200
+        )
+        return factory(DowcResponse, response)
```

Comment:
you can keep this as is - but the `gemma-zds-client` exposes the requests interface (see https://github.com/VNG-Realisatie/gemma-zds-client/blob/master/zds_client/client.py#L181) so the cleanest approach would probably be:

```suggestion
    query = {
        "purpose": DocFileTypes.write,
        "info_url": referer,
    }
    try:
        response = client.request(
            url, operation_id, method="GET", expected_status=200, params=query
        )
        return factory(DowcResponse, response)
```